docs/development/security/gdpr_regulation.md

```diff
@@ -0,0 +1,156 @@
+
+The personal information processed in the system is data such as the name of the citizen (elaborate with any other information), along with the unique calendar assigned.
+
+### Essential Principles
+
+The GDPR  regulation states several essential principles, by which every data controller should ensure their business processes operate. Based on the previous discussion of definitions, the primary responsibilities falls upon the institutions adopting the GIRAF system, but in order for any organisation to be willing to do so, the GIRAF system must be designed and implemented with the regulation in mind, which is therefore the responsibility of the development team.
+
+The principles is summarized to the following list of tasks for the GIRAF project:
+* Determine what data is sufficient to fulfill the purpose of the purpose of the GIRAF system, and document both the details of the data, how and when it is collected and for which purpose.
+* Determine or define how the correctness of the data is ensured, and document it.
+* Define reasonable step to take, to ensure the integrity, confidentiality and security of the personal data, which applies both to organisational and technical levels.
+Determine for how long data has to be retained, for it to fulfill its purpose, along with both when and by what means, it should be deleted.
+
+### Legal Processing
+
+For any processing of personal data to be legal, there has to be appropriate lawful purpose to do so. There is a number of lawful purposes for processing personal data, one of which is the use of consent from the data subject, by means of a consent contract for example. The GDPR regulation defines several stipulations, regarding obtaining consent from the data subject, such as requiring high privacy and data protection by default, and any request for data collection to have a specific purpose and be of opt-in nature. Further, any processing of data regarding children, requires consent of the parent or guardian of the child. These requirements should be considered in the tasks regarding the GIRAF project.
+* Determine the lawful basis on which personal data is to be processed in the GIRAF system.
+    * Based on this, determine the most appropriate means of obtaining such lawful basis, such as seeking to establish a consent contract with the data subject.
+        * Should the data subject be a child, determine how the consent should be obtained from the parents or guardian.
+
```

Comment:
I think you should consider remaking this. My issue with it is that its like you are constructing a paragraph of sentences by indenting the bulleted points. I suggest to maybe just write it as normal text, as it is kind of already. Or consider rephrasing the bullets a bit.